app/src/utils.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { GetDefaultRoute } from "./utils";
+
+test('GetDefaultRoute always returns default route name', () => {
+    const str1 = GetDefaultRoute();
+
+    expect(str1).toEqual("rate-my-cse");
+    expect(str1).toEqual(GetDefaultRoute());
```

Comment:
This line is redundant. You can trust that str1 is equal to GetDefaultRoute()

Reply:
removed